<launch>
	<arg name="odom" />
	<arg name="position_cmd" />
	<arg name="mandatory_stop" />
	<arg name="cancel_mandatory_stop"/>
	
	<node pkg="px4ctrl" type="px4ctrl_node" name="px4ctrlcoax" output="screen">
        	<!-- <remap from="~odom" to="/odom_correction/odom" /> -->
        	<remap from="~odom" to="$(arg odom)" />
			<remap from="~cmd" to="$(arg position_cmd)" />
			<remap from="/mandatory_stop" to="$(arg mandatory_stop)" />
			<remap from="/cancel_mandatory_stop" to="$(arg cancel_mandatory_stop)"/>
        	<rosparam command="load" file="$(find px4ctrl)/config/ctrl_param_coax.yaml" />
	</node>
</launch>